lib/net.js

```diff
@@ -580,6 +585,9 @@ Socket.prototype._getpeername = function() {
   return this._peername;
 };
 
+Socket.prototype.__defineGetter__('bytesRead', function() {
```

Comment:
Since `__defineGetter__` is deprecated now, can these be turned into proper `Object.defineProperty()` calls? Perhaps as a separate commit in this PR?

e.g.
```
Object.defineProperty(Socket.prototype, 'bytesRead', {
  configurable: false,
  enumerable: true,
  get: function() {
    return this._handle ? this.handle.bytesRead : this[BYTES_READ];
  }
});
```